OpenRA.Game/ModData.cs

```diff
@@ -74,10 +74,8 @@ public ModData(string mod, bool useLoadScreen = false)
 			SpriteSequenceLoader = (ISpriteSequenceLoader)ctor.Invoke(new[] { this });
 			SpriteSequenceLoader.OnMissingSpriteError = s => Log.Write("debug", s);
 
-			// HACK: Mount only local folders so we have a half-working environment for the asset installer
+			// We cannot put GlobalFileSystem.LoadFromManifest(Manifest) here because Travis doesn't like that
 			GlobalFileSystem.UnmountAll();
```

Comment:
We probably don't need this or the comment above anymore.  The reasonable default assumption is that constructors don't have ugly side-effects on global state.